src/commands/kubeval.yaml

```diff
@@ -6,9 +6,11 @@ description: Validate Kubernetes resources.
 steps:
   - run:
       name: kubeval
+      environment:
+        KUBEVAL_SCHEMA_LOCATION: https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master
```

Comment:
is pointing at `master` a good idea? What if it changes and suddenly all builds break because of a change we don't know of?

Reply:
Good point, but this repository has 1 branch, 0 tags, the best I can do is point to current master commit `411622f5e91bc54d3eff7b700661ba03f23f7729`. Does this works for you?